pagopa/io-app
# chore(IT Wallet): [SIW-1287] Update credential preview screens

## Short description
This PR updates the credentials preview screen, removing the card preview.

## List of changes proposed in this pull request
- Removed card preview from `ItwIssuanceCredentialPreviewScreen` component
- Removed card preview from `ItwIssuanceEidPreviewScreen` component
- Deleted `ItwCredentialPreviewScreenContent` component
- Updated `ItwIssuanceEidResultScreen`
- Updated locales

## How to test
From the **IT Wallet playgrounds**, navigate to the **eID** and **mDL** issuance flows and check that the preview screens are rendered correctly.

## Preview

https://github.com/pagopa/io-app/assets/6160324/2b7a4c1f-8de3-49ea-85cd-5b5c3ed4c0ab
